wordpress.org/public_html/wp-content/themes/pub/wporg-plugins/header.php

```diff
@@ -19,7 +19,21 @@
 	'/developers/'       => __( 'Developers', 'wporg-plugins' ),
 );
 
-echo do_blocks( '<!-- wp:wporg/global-header /-->' );
+$local_nav_items = array(
+	'' => __( 'All', 'wporg-plugins' ),
+	'community' => __( 'Community', 'wporg-plugins' ),
+	'commercial' => __( 'Commercial', 'wporg-plugins' ),
+);
+
+global $wp_query;
+$is_beta = 'beta' === $wp_query->get( 'browse' );
+// The filter bar should not be shown on:
+// - search: wp-query filters not compatible with ES search.
```

Comment:
This is unfortunate. I would expect it to be most valuable on the search page since the browsing the directory isn't a great experience. 

During the redesign, we should make sure to get this working or at least scope.

Reply:
Sounds good.